src/main/java/org/threadly/litesockets/utils/MergedByteBuffers.java

```diff
@@ -17,13 +17,13 @@
  */
 public class MergedByteBuffers {
```

Comment:
ByteBuffers are not even close to thread safe.  I guess they would not throw a concurrentMod Exception like this might, but its prettty impossible to use the same ByteBuffer instance concurrently and reliably.